SCClassLibrary/Common/GUI/Base/QView.sc

```diff
@@ -89,7 +89,7 @@ View : QObject {
 		this.background = color;
 	}
 
-	backgroundImage_ { arg image; this.setBackgroundImage(image) }
+ 	backgroundImage_ { arg image; this.setBackgroundImage(image) } 
```

Comment:
this change shouldn't be here.. it looks like all you really need to do for this PR is make the single change to the other line in `QImage.sc`. I would recommend making that change and force pushing to update this branch, then making the gitignore PR separately. If you want help with that I can give more advice here, via email or slack.